integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisApplicationIntegrationTest.java

```diff
@@ -371,8 +371,11 @@ public void testIcebergCreateTablesInExternalCatalog() throws IOException {
                           TableIdentifier.of(ns, "the_table"),
                           new Schema(
                               List.of(
-                                  Types.NestedField.of(
-                                      1, false, "theField", Types.StringType.get()))))
+                                  Types.NestedField.builder()
+                                      .withId(1)
+                                      .withName("theField")
+                                      .ofType(Types.StringType.get())
+                                      .build())))
```

Comment:
Did you avoid `Types.NestedField.required()` method intentionally?

Reply:
`asRequired()` you mean? That's the default, so I indeed omitted that. Do you prefer to explicitly call `asRequired()` each time?

Reply:
I meant this helper method: https://github.com/apache/iceberg/blob/b0596267abd0d474b00ff41d963244fdaffc08dc/api/src/main/java/org/apache/iceberg/types/Types.java#L712-L714

```java
    public static NestedField required(int id, String name, Type type) {
      return new NestedField(false, id, name, type, null, null, null);
    }
```